datashuttle/configs/config_class.py

```diff
@@ -121,44 +122,14 @@ def load_from_file(self) -> None:
         with open(self.file_path, "r") as config_file:
             config_dict = yaml.full_load(config_file)
 
-        self.convert_str_and_pathlib_paths(config_dict, "str_to_path")
+        load_configs.convert_str_and_pathlib_paths(config_dict, "str_to_path")
 
         self.data = config_dict
 
     # -------------------------------------------------------------------------
     # Utils
     # -------------------------------------------------------------------------
 
-    def convert_str_and_pathlib_paths(
```

Comment:
This is a refactor, moving to another module to avoid the strange signature that was occuring:

```configs.convert_str_and_pathlib_paths(configs, ...)```